spec/factories/clients.rb

```diff
@@ -1,4 +1,7 @@
 FactoryBot.define do
   factory :client do
+    email { 'client@email.com' }
+    password { 'password' }
+    cpf { '082.923.869-71' }
```

Comment:
Aqui daria pra usar um:

```suggestion
    cpf { CPF.generate(formatted: true) }
```

Pelo mesmo motivo do email, o que acha?